app/src/main/java/app/revanced/integrations/videoplayer/ExternalDownloadButton.java

```diff
@@ -49,7 +49,7 @@ private static void onDownloadClick(View view) {
         LogHelper.printDebug(() -> "External download button clicked");
 
         final var context = view.getContext();
-        var downloaderPackageName = SettingsEnum.EXTERNAL_DOWNLOADER_PACKAGE_NAME.getString();
+        var downloaderPackageName = SettingsEnum.EXTERNAL_DOWNLOADER_PACKAGE_NAME.getString().trim();
```

Comment:
```suggestion
        // Trim string to avoid any accidental whitespace.
        var downloaderPackageName = SettingsEnum.EXTERNAL_DOWNLOADER_PACKAGE_NAME.getString().trim();
```

Reply:
Oh yeah I guess it would make sense to add a comment for that, I apologize it's 11 PM